sdk/search/azure-search-documents/azure/search/documents/_service/_search_index_client.py

```diff
@@ -55,11 +54,23 @@ def __exit__(self, *args):
 
     def close(self):
         # type: () -> None
-        """Close the :class:`~azure.search.documents.SearchIndexesClient` session.
+        """Close the :class:`~azure.search.documents.SearchIndexClient` session.
 
         """
         return self._client.close()
 
+    @distributed_trace
```

Comment:
Does this need to be here? creating a `SearchClient` should not make any API calls